src/btree/bt_handle.c

```diff
@@ -323,7 +323,8 @@ __btree_conf(WT_SESSION_IMPL *session, WT_CKPT *ckpt, bool is_ckpt)
         maj_version = cval.val;
         WT_RET(__wt_config_gets(session, cfg, "version.minor", &cval));
         min_version = cval.val;
-        __wt_verbose(session, WT_VERB_VERSION, "%" PRId64 ".%" PRId64, maj_version, min_version);
+        __wt_verbose(
+          session, WT_VERB_VERSION, "btree version:%" PRId64 ".%" PRId64, maj_version, min_version);
```

Comment:
NP: Can we add a space here: "btree version: %", before "%" please.

Reply:
done, thanks, please review again.